.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
```

Comment:
This should be closer to https://github.com/apache/flink-connector-elasticsearch/blob/main/.idea/vcs.xml for some nifty IDE features.